### PR TITLE
Tweaks to publish and builds

### PR DESCRIPTION
**Pull request by mattgrill**
An interesting note about Babel 6,

Currently your built copy looks like, 

```javascript
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

exports.default = function () {
  return 'Yes';
};
```

Which results in some nasty import code, 

```javascript
const ted = require('istedchill').default;
```
_Lets fix that_,

```javascript
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

exports.default = function () {
  return 'Yes';
};

module.exports = exports['default'];
```

Which allows you to do this, 

```javascript
const ted = require('istedchill');
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,12 +8,25 @@
     "url": "https://github.com/dawehner/istedchill.git"
   },
   "scripts": {
-    "prepublish": "npm run build",
+    "prepublishOnly": "npm run build",
```
Comment (dawehner) [inline package.json:11]: aaaaaaaah

Comment (dawehner) [reply package.json:11]: Nice

**Comment by dawehner** [comment]
Good to learn!

**Comment by mattgrill** [comment]
@dawehner Have a look, https://docs.npmjs.com/misc/scripts#prepublish-and-prepare

**Pull request merged**